src/inspect_ai/tool/_tool_call.py

```diff
@@ -31,6 +31,8 @@ class ToolCallError:
         "permission",
         "file_not_found",
         "is_a_directory",
+        "output_limit",
```

Comment:
Again would be in favor of a single "output_limit" here as "read_file_limit" is very specific.